native/core/src/execution/planner.rs

```diff
@@ -1153,7 +1161,7 @@ impl PhysicalPlanner {
                 let child_copied = Self::wrap_in_copy_exec(Arc::clone(&child.native_plan));
 
                 let sort = Arc::new(
-                    SortExec::new(LexOrdering::new(exprs?), Arc::clone(&child_copied))
+                    SortExec::new(LexOrdering::new(exprs?).unwrap(), Arc::clone(&child_copied))
```

Comment:
note for other reviews: the `unwrap()` here is on an `Option`. This seems safe because it would only be `None` if the vector of sort expressions were empty.